storybookjs/storybook
# fix info not importing docgen descriptions

Issue:

Addon-Info no longer displays component description comment from react-docgen. Think it might have been broken since we implemented hierarchy.

![screen shot 2017-10-26 at 6 44 41 pm](https://user-images.githubusercontent.com/1474548/32080488-ed04c72a-ba7d-11e7-8967-d1a53cf32fb9.png)

## What I did

fixed it

## How to test

Is this testable with jest or storyshots?
yes

Does this need a new example in the kitchen sink apps?
no, existing example should already work

Does this need an update to the documentation?
yes

If your answer is yes to any of these, please make sure to include it in your PR.
